frontend/ui/widget/textboxwidget.lua

```diff
@@ -367,6 +590,34 @@ function TextBoxWidget:_renderText(start_row_idx, end_row_idx)
     self._bb = Blitbuffer.new(self.width, h, bbtype)
     self._bb:fill(Blitbuffer.COLOR_WHITE)
     local y = font_height
+
+    if self.use_xtext then
+        for i = start_row_idx, end_row_idx do
+            local line = self.vertical_string_list[i]
+            self:_shapeLine(line)
+            if line.xglyphs then -- non-empty line
+                for i, xglyph in ipairs(line.xglyphs) do
+                    local face = self.face.getFallbackFont(xglyph.font_num) -- callback (not a method)
+                    local glyph = RenderText:getGlyphByIndex(face, xglyph.glyph, self.bold)
+                    local color = self.fgcolor
+                    if self._alt_color_for_rtl then
```

Comment:
So this is always disabled unless you change it in the code?

Reply:
(Might be of some interest in that developer settings menu.)

Reply:
> So this is always disabled unless you change it in the code?

Right, always disabled.
Not worth busying developper settings with it I think - was useful when doing this PR, but I guess RTL is alright now, and I shouldn't need it - unless we feel it's needed when we play with mirroring the UI, but I doubt it.